test-apps/ui-test-app/README.md

```diff
@@ -0,0 +1,131 @@
+3. Optionally, set other environment variables to configure the application prior to startup. The full list of supported variable are [below](#environment-variables).
+
+* To start the application in Electron, navigate to the root of ui-test-app, and use the command:
+
+  ```cmd
+  npm run start
+  ```
+
+* To start the application in a browser, run the following command, and then navigate to localhost:3000 in any supported browser:
+
+  ```cmd
+  npm run start:servers
+  ```
+
+* To start the application as an iOS app, run the following: WIP
+
+## Dependencies
+
+* Installed dependencies for ui-test-app may be found in the generated node_modules directory. Since ui-test-app is but a part of a larger monorepo, the dependencies here are provided as symlinks into a master node_modules directory managed by the build tool Rush.
+* Any changes made to itwinjs-core files outside of this directory will not immediately be reflected in ui-test-app. The entire itwinjs-core monorepo must be rebuilt in order for changes to take effect.
+* If dependencies have changed after pulling the most recent version of itwinjs-core, it is often necessary to do a clean reinstall of all dependencies in order to avoid build errors.
+
+```cmd
+rush install -c
+```
+
+## Environment Variables
+
+You can use these environment variables to alter the default behavior of various aspects of the application.
+
+To set the environment variables, either set them directly within the terminal you start the test app or create a `.env` file as a peer to this README.
+
+### Common variables
+
+* IMJS_UITESTAPP_IMODEL_NAME
+  * (Online-only) Set the iModel to open when the app starts. The iTwin Name variable is required for this one to properly work and the iModel to be opened
+* IMJS_UITESTAPP_ITWIN_NAME
+  * (Online-only) Set the iTwin to open when the app starts. The iModel Name variable is required for this one to properly work and the iModel to be opened.
+* IMJS_UITESTAPP_IMODEL_VIEWID
+  * (Online-only) Opens the provided iModel to the viewId, otherwise IModelIndex page is used.
+* IMJS_UITESTAPP_SNAPSHOT_FILEPATH
+  * Set to folder containing .bim or .ibim files (required)
+* IMJS_UITESTAPP_START_WITH_SNAPSHOTS
+  * Set to 1 to start with File Open dialog (optional)
+* IMJS_UITESTAPP_ALLOW_WRITE
+  * Editing Support - set to 1. If not defined, false is assumed. WARNING: this is strictly used to test the UI when editing is enabled
+* IMJS_UITESTAPP_USE_LOCAL_SETTINGS
+  * Store settings in LocalStorage instead of Project Settings Service
+
+* IMJS_MAPBOX_KEY
+  * If defined, sets the MapBox key for the `MapLayerOptions` as an "access_token".
+* IMJS_BING_MAPS_KEY
+  * If defined, sets a Bing Maps key within the `MapLayerOptions` as a "key" type.
+* IMJS_CESIUM_ION_KEY
+  * If defined, the API key supplying access to Cesium ION assets.
+* IMJS_URL_PREFIX
```

Comment:
This is the only variable that I think would confuse other users as they would not know what to set this to or why they would need to set it. Should you be more specific like it specifies to log into QA DEV or PROD?

Reply:
removed that section for now. We can add it back later if needed.